careers/_vars-careers.html

```diff
@@ -0,0 +1,11 @@
+{% set path = '/careers/' %}
+{% set nav_items = [
+    (path, 'index', 'Careers at the CFPB'),
+    (path + 'working-at-cfpb/', 'working-at-cfpb', 'Working at CFPB'),
+    (path + 'application-process/', 'application-process', 'Job Application Process'),
+    (path + 'students-and-graduates/', 'students-and-graduates', 'Students & Graduates'),
```

Comment:
Confirm that it should be `Students & Graduates` and not `Students and Graduates` like the page title.